portal-kernel/src/com/liferay/document/library/kernel/util/packageinfo

```diff
@@ -1 +1 @@
-version 7.0.0
\ No newline at end of file
+version 7.1.0
```

Comment:
Can you check the 
`bnd.bnd`of the portal-kernel

i think a upgrade bundle-version is needed > `Bundle-Version: 67.1.0`

Reply:
let me check it 
Thank @AliciaGarciaGarcia

Reply:
[semantic-versioning-jdk8/0/0](https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/83346//consoleText)
clean-up-java-processes:
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/clean_up_java_processes.sh
[stopwatch] [run.batch.test.setup: 29:15.309 sec]
     [echo] run.batch.test.action.start.timestamp: 08-16-2022 06:26:49:382 PDT
     [echo] Checking for baseline log files.
     [echo] 
     [echo] ##
     [echo] ## /opt/dev/projects/github/liferay-portal/baseline-reports/portal-kernel.log
     [echo] ##
     [echo] 
[beanshell] [Baseline Warning] Bundle Version Change Recommended: 67.1.0